.github/workflows/book.yml

```diff
@@ -17,8 +17,21 @@ jobs:
           python-version: 3.9
       - name: Install Poetry
         uses: snok/install-poetry@v1.3
+      - name: Cache Poetry virtualenv
+        uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/.virtualenvs
+          key: poetry-${{ hashFiles('**/poetry.lock') }}
+          restore-keys: |
+            poetry-${{ hashFiles('**/poetry.lock') }}
+      - name: Set Poetry config
+        run: |
+          poetry config virtualenvs.in-project false
```

Comment:
Isn't that the default? https://python-poetry.org/docs/configuration/#virtualenvsin-project